pkg/proxy/endpointslicecache.go

```diff
@@ -164,7 +164,7 @@ func (cache *EndpointSliceCache) addEndpointsByIP(serviceNN types.NamespacedName
 		}
 
 		isLocal := cache.isLocal(endpoint.Topology[v1.LabelHostname])
-		endpointInfo := newBaseEndpointInfo(endpoint.Addresses[0], portNum, isLocal)
+		endpointInfo := newBaseEndpointInfo(endpoint.Addresses[0], portNum, isLocal, "")
```

Comment:
We don't need nodeName if switched to watch EndpointSlice, so leave a "" to it, will remove nodeName when we switched to watch EndpointSlice to get topology infomation.

Reply:
Add a TODO comment where nodeName was added?